src/main/java/de/diddiz/util/TypeColor.java

```diff
@@ -0,0 +1,29 @@
+package de.diddiz.util;
+
+import org.bukkit.ChatColor;
+
+public enum TypeColor {
+    DEFAULT(ChatColor.YELLOW),
+    MATERIAL(ChatColor.BLUE),
+    STATE(ChatColor.BLUE),
+    DATE(ChatColor.DARK_AQUA),
+    BRACKETS(ChatColor.DARK_GRAY),
+    COORDINATE(ChatColor.WHITE),
+    HEADER(ChatColor.GOLD),
+    ERROR(ChatColor.RED);
```

Comment:
Possible improvement is making these color code constants loaded from a config file rather than hard coded, or alternatively replacing these with pattern/templated configurable strings.